test/test_estimators/test_estimators.py

```diff
@@ -378,6 +378,74 @@ def test_cv_results(tmp_dir):
     assert hasattr(cls, "classes_")
 
 
+def test_cv_results_multi_objective(tmp_dir):
+    # TODO restructure and actually use real SMAC output from a long run
+    # to do this unittest!
+    X_train, Y_train, X_test, Y_test = putil.get_dataset("iris")
+
+    cls = AutoSklearnClassifier(
+        time_left_for_this_task=30,
+        per_run_time_limit=5,
+        tmp_folder=os.path.join(tmp_dir, "backend"),
+        seed=1,
+        initial_configurations_via_metalearning=0,
+        metric=[autosklearn.metrics.precision_macro, autosklearn.metrics.roc_auc],
+        scoring_functions=[autosklearn.metrics.accuracy, autosklearn.metrics.roc_auc],
+    )
+
+    params = cls.get_params()
+    original_params = copy.deepcopy(params)
+
+    cls.fit(X_train, Y_train)
+
+    cv_results = cls.cv_results_
+    assert isinstance(cv_results, dict), type(cv_results)
+    assert "mean_test_score" not in cv_results
+    assert "rank_test_scores" not in cv_results
+    for expected_column in (
+        "mean_test_precision_macro",
+        "mean_test_roc_auc",
+        "mean_fit_time",
+        "rank_test_precision_macro",
+        "rank_test_roc_auc",
+        "metric_roc_auc",
+        "metric_accuracy",
+    ):
+        assert isinstance(cv_results[expected_column], np.ndarray), type(
```

Comment:
[Just a comment]
Hopefully as we can get to making more things typed, for example using typed dicts, we can eliminate the need to manually type check things.